### DISCUSSION TITLE
why image not generat?

### DISCUSSION THREAD

**Discussion by nimafar83** [vladmandic/sdnext#3131] [Q&A]
i write  prompt and click start  but nothing happend

![Screenshot 2024-05-14 165054](https://github.com/vladmandic/automatic/assets/131419563/55820c7b-d7a9-4d84-a2dd-22fea748c002)

**Comment by brunogcar** [comment]
try to install on a normal folder on your driver instead of desktop, like C:\Automatic\
everything on desktop is under Users which have different set of permissions

**Reply by nimafar83** [reply]
i do
dont work

**Comment by Disty0** [comment]
It is using the CPU, what GPU do you have?

**Reply by nimafar83** [reply]
intel(R) iris(R) Xe

**Reply by Disty0** [reply]
1) You don't have a proper GPU, expect things to be very slow.
2) You don't have enough RAM. 8 GB is not enough.
